### PR TITLE
fix: only initiate WC v2 module if project id provided

### PR DESCRIPTION
**Pull request by iamacook**
## What it solves

Marks `NEXT_PUBLIC_WC_PROJECT_ID` as required

## How this PR fixes it

If `NEXT_PUBLIC_WC_PROJECT_ID` is not present, the WC v2 module throws.

The module is now not instantiated if one is not present and the README has been updated accordingly.

## Checklist
* [ ] I've tested the branch on mobile 📱
* [ ] I've documented how it affects the analytics (if at all) 📊
* [ ] I've written a unit/e2e test for it (if applicable) 🧑‍💻

### CODE REVIEW

**Review by katspaugh — COMMENTED** [review]
I would avoid adding all that white space and trailing `|`'s as they are optional.

**Comment by iamacook** [comment]
> I would avoid adding all that white space and trailing `|`'s as they are optional.

This was added by Prettier. I'll disable it and push again.

File: README.md [hunk 1]
```diff
@@ -22,13 +22,13 @@ Here's the list of all the required and optional variables:
 | Env variable                                           |              | Description
 | ------------------------------------------------------ | ------------ | -----------
 | `NEXT_PUBLIC_INFURA_TOKEN`                             | **required** | [Infura](https://docs.infura.io/infura/networks/ethereum/how-to/secure-a-project/project-id) RPC API token
+| `NEXT_PUBLIC_WC_PROJECT_ID`                            | **required** | [WalletConnect v2](https://docs.walletconnect.com/2.0/cloud/relay) project ID
```
Comment (katspaugh) [inline README.md:25]: It's not required then, is it?

**Review by katspaugh — APPROVED** [review]

**Comment by francovenica** [comment]
LGTM

Talked with Aaron to see how to reproduce the issue and see the fix. 
Checked in my local dev and then in this branch. In dev the app crashes, in this PR it doesnt.

In dev this was happening:
image

**Pull request merged**